controllers/comment.js

```diff
@@ -0,0 +1,25 @@
+const commentService = require('~/services/comment')
+
+const create = async (req, res) => {

```

Comment:
createComment
follow namings in other controllers

Reply:
for me it looks clearer but okay, i will change